index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html  ng-app="myApp">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        
+        <title>Library Information System Angular 1</title>
+        
+        <!-- Bootstrap CSS CDN -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
we will need to host the files ourselves so we can ensure the versions won't change. Can you add a task to remove all these cdn links and have relative links to downloaded versions?